docs/site/DEVELOPING.md

```diff
@@ -1,3 +1,11 @@
+---
+lang: en
+title: 'Contributing code in LoopBack 4'
+keywords: LoopBack 4.0, contributing, community
```

Comment:
Is "LoopBack 4.0" a known keyword? I mean do we need to register it somewhere in a YAML file?

Reply:
Looks like most (if not all) docs pages are using `LoopBack 4.0`.  I searched for this term and 86 occurrences.  
I didn't find any instructions about registering the keywords.  Do you know how I can do that?